fuzzers/run.sh

```diff
@@ -3,6 +3,8 @@
 set -eu
 
 TARGET=${1-"rtcp_fuzzer"}
+JOBS=${JOBS:-4}
+WORKERS=${WORKERS:-4}
```

Comment:
I'll let @atoppi comment on the PR in general, but why is the value `-4` there? Shouldn't it be `4`?

Reply:
@lminiero in bash, that's the syntax to specify default values if missing, it's dash 4, not minus 4.

Reply:
Thanks for the clarification, I'm an idiot :smile:

Reply:
@lminiero not at all, I have to look this up every time ;)